src/env/env.rs

```diff
@@ -70,8 +70,9 @@ fn parse_program_opt<'a>(opts: &mut Options<'a>, opt: &'a str) -> Result<(), i32
 }
 
 fn load_config_file(opts: &mut Options) -> Result<(), i32> {
-    // NOTE: config files are parsed using an INI parser b/c it's available and compatible with ".env"-style files
-    //   ... * but support for actual INI files, although working, is not intended, nor claimed
+    // NOTE: config files are parsed using an INI parser b/c it's available and
+    // compatible with ".env"-style files   ... * but support for actual INI
```

Comment:
Does `rustfmt` have a different line length for comments?  This is really bizarre IMO.

Reply:
The above line is exactly `80` characters, perhaps GitHub formatting is weird?

Reply:
What I mean is that the standard line length for code is 100 characters (or it used to be at least), but for comments it seems shorter?  Or maybe it's just some comments?  Not really sure.

Reply:
Yeah, code is `100` characters but commens `80`.